src/app/layout.tsx

```diff
@@ -18,9 +24,15 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={PlusJakarta.className}>
-        <MuiThemeRegistry>{children}</MuiThemeRegistry>
+        <MuiThemeRegistry>
+          <NavigationDrawer width={DRAWER_WIDTH} />
+          <Box sx={{ ml: { sm: `${DRAWER_WIDTH}px` } }}>
+            <PagesTop />
+            {children}
+          </Box>
+        </MuiThemeRegistry>
 
-        <Script src="/scripts/grafana.js" />
+        <Script src="/public/scripts/grafana.js" />
```

Comment:
Grafana is causing some trouble with Next.js `Link`. Need to discuss with @mehrdadmms